PyInstaller/utils/win32/winmanifest.py

```diff
@@ -269,8 +269,21 @@ def __init__(self, manifestVersion=None, noInheritable=False,
         self.bindingRedirects = []
         self.files = files or []
         self.comInterfaceExternalProxyStubs = comInterfaceExternalProxyStubs or [] # TO-DO: implement
-    
-    def add_dependent_assembly(self, manifestVersion=None, noInheritable=False, 
+
+    def __eq__(self, other):
+        if isinstance(other, Manifest):
+            return self.toxml() == other.toxml()
+        if isinstance(other, string_types):
+            return self.toxml() == other
+        return False
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
+    def __repr__(self):
+        return "'%s'" % self.toxml()
```

Comment:
I'm afraid, this will fail if `self.toxml()` contains a single quote. I suggest using `repr(self.toxml())`.